ext/dom/xpath.c

```diff
@@ -132,7 +137,7 @@ static void dom_xpath_ext_function_php(xmlXPathParserContextPtr ctxt, int nargs,
 	if (obj->stringval == NULL) {
```

Comment:
Maybe add a comment that this `obj` is the last element of the FILO stack and must correspond to a PHP function name?

Reply:
Ack